lib/src/mt_dma.c

```diff
@@ -480,6 +480,7 @@ uint16_t mt_dma_completed(struct mtl_dma_lender_dev* dev, uint16_t nb_cpls,
 int mt_dma_borrow_mbuf(struct mtl_dma_lender_dev* dev, struct rte_mbuf* mbuf) {
   struct mt_dma_dev* dma_dev = dev->parent;
 
+  rte_mbuf_refcnt_update(mbuf, 1);
   st_rx_mbuf_set_lender(mbuf, dev->lender_id);
 #if MT_DMA_RTE_RING
   int ret = rte_ring_sp_enqueue(dma_dev->borrow_queue, (void*)mbuf);
```

Comment:
Do we need dec refcnt if rte_ring_sp_enqueue fail here?

Reply:
> Do we need dec refcnt if rte_ring_sp_enqueue fail here?

good point, the update should be end of func